src/pages/namespace/Explorer/Endpoint/EndpointEditor/schema/plugins/inbound/index.ts

```diff
@@ -2,4 +2,17 @@ export const inboundPluginTypes = {
   acl: "acl",
   jsInbound: "js-inbound",
   requestConvert: "request-convert",
+  eventFilter: "event-filter",
 } as const;
+
+const allPlugins = Object.values(inboundPluginTypes);
+const enterpriseOnlyPlugins = [inboundPluginTypes.eventFilter] as string[];
```

Comment:
I think this way of flagging EE only plugins isn't ideal. It may be acceptable at this point because we have relatively few plugins.

But what would you think about extending the structure of `targetPluginTypes` to include more information?

Something like:
```
export const targetPluginTypes = {
  instantResponse: { name: "instant-response", enterpriseOnly: false },
  ..
} as const;

I have no idea if this would make things complicated on the typescript front. But in general, I would find it cleaner to have only one list containing all the information.

Reply:
I think this is a very good idea. I had something like this in my mind as well, but did not want to change the structure too much. But I just reviewed it and find it too complicated to read for such a simple filtering, so I did some refactoring like you suggested. It would be great if you can take a final look [at my changes](https://github.com/direktiv/direktiv-ui/compare/cc00fd9f8c1430d01e2a9896bbacadf2e4460115...1c1184b9c6d72e68f0e4a47e9051d885d94e5a78)

Reply:
FYI, target is now `main` since the old target was merged into main `stefankracht/dir-1027-make-403-redirect-to-logout-path `